.vscode/launch.json

```diff
@@ -40,13 +40,30 @@
           "kind": "bin"
         }
       },
+      "args": [
+        "--kernel",
+        "${workspaceFolder}/build/obkrnl"
+      ],
       "cwd": "${workspaceFolder}"
     },
     {
-      "name": "Kernel",
+      "name": "Kernel (Qt)",
```

Comment:
The parentheses here does not look good on the VS Code due to VS Code also put some text after this with parentheses. Better to change to another format (e.g. `Kernel | Qt`).